tdm-reader/nitdmreader.yml

```diff
@@ -0,0 +1,512 @@
+    description: An operation provided by the API
+    type: object
+    properties:
+      available:
+        type: boolean
+        description: Whether the operation is available to the caller
+      version:
+        type: integer
+        description: The version of the available operation
+    required: [available]
+    example:
+      available: true
+      version: 1
+  V1Operations:
+    description: V1 operations
+    type: object
+    properties:
+      operations:
+        description: >-
+          Available operations in the v1 version of the API:
+
+          - readData: The ability to read data
+        type: object
+        properties:
+          readData:
+            $ref: '#/definitions/Operation'
+  Property:
+    description: A property and its value.
+    type: object
+    properties:
+      name:
+        type: string
+        description: name of the property
+        example: Name
+      type:
+        type: string
+        description: data type of 'value'
+        example: String
+      value:
+        type: string
+        example: Channel name
+  Properties:
+    type: object
+    properties:
+      properties:
+        description: Property values.
+        type: array
+        items:
+          $ref: '#/definitions/Property'
+      totalCount:
+        type: integer
+        description: Count of all properties. This could be more than the items returned in Properties.
+        example: 1
+  FileList:
+    type: object
+    properties:
+      fileIds:
+        description: File Ids
+        type: array
+        items:
+          type: string
+          example: 5b8453dba8808c22b0f99023
+  DataWindow:
+    description: Data window.
+    properties:
+      decimatex:
+        type: integer
+        description: Number of pixels on the x axis which is the number of points that can be displayed.
+        example: 1000
+        default: 0
+        format: int32
+      decimatey:
+        type: integer
+        description: Number of pixels on the y axis which is the number of points that can be displayed
+        example: 1000
+        default: 0
+        format: int32
+      xmin:
+        type: number
+        default: 0
+        format: double
+        description: Minimum X value of the display X axis.
+        example: -1
+      xmax:
+        type: number
+        default: 0
+        format: double
+        description: Maximum X value of the display X axis.
+        example: 1
+      ymin:
+        type: number
+        default: 0
+        format: double
+        description: Minimum Y value of the display Y axis.
+        example: -1
+      ymax:
+        type: number
+        default: 0
+        format: double
+        description: Maximum Y value
+        example: 1
+      showLines:
+        type: boolean
+        description: Decimate to line style graph.
+        default: true
+      showPoints:
+        type: boolean
+        default: false
+        description: Decimate to point style graph.
```

Comment:
Here and in the previous description, I'm not sure even after some googling what decimate means in this context. Is there a way we could put this in plainer terms for newer users? "Represent data as a point-style graph" or something?